app/models/ep_postmaster/mailgun_post.rb

```diff
@@ -2,24 +2,31 @@
 
 module EpPostmaster
   class MailgunPost
-    attr_accessor :message_id, :x_mailgun_sid, :code, :message_headers, :domain, :error, :event, :recipient, :reply_to, :subject, :signature, :timestamp, :token, :from, :sender
+
+    attr_accessor :message_id, :code, :message_headers, :domain, :error, :event, :recipient, :reply_to, :subject, :signature, :timestamp, :token, :from, :sender, :reason
 
```

Comment:
This was refactored to use the new webhooks in place of the legacy webhooks.  See PR description for details.